### ISSUE TITLE
suggestion:some utils functions should be replaced

### ISSUE THREAD

**Issue by guzuomuse** [arkon/ng-sidebar#85]
for example: the function isBrowser() comes from utils.js should be replace with the angular build in method isPlateformBrowser(...);
the reason is :  if this libary used form "universal"  as know as 'server side rendering' there are some errors such as "Windows is not defined";
any suggestions? thank you.

**Comment by arkon** [comment]
`isBrowser()` is just performing a `typeof` check, so if it doesn't exist (i.e. isn't a browser, so something like SSR), then it would just return `false`. This function is used as a guard in several places and (should) work fine.

I didn't really want to use Angular's `isPlatformBrowser()` since it involves injecting more things with no real benefit (just introduces more indirection and bloat in my opinion).

**Comment by guzuomuse** [comment]
@arkon ,but there will some errors  when used "universal".
"Windows is not defined".
have you tested for "universal" usage?

**Comment by arkon** [comment]
@guzuomuse I haven't, no. Are you saying that it _does_ give that error, or that you _think_ it will?

**Comment by guzuomuse** [comment]
it does!

**Comment by arkon** [comment]
@guzuomuse Ah sorry, I misunderstood you. I'll fix it then.

**Comment by guzuomuse** [comment]
@arkon  never mind   :) thanks for your effort!

**Comment by guzuomuse** [comment]
@arkon ,just a remind ,i think all the client's variables should be took care.  such as `window` `navigator ` 
looking for the next new release......

**Comment by arkon** [comment]
@guzuomuse The library depends on those to work, so the most I can do is to prevent any of it from really running on the server.

**Comment by guzuomuse** [comment]
@arkon ,hi glad to see the new release!

But there are still some nimor flaws lead to the "universal " app can't to run.

here is the workaroud in utils.ts

```
export function isLTR(): boolean {
  let dir: string = 'ltr';

if (typeof window !== 'undefined') {
    if (window.getComputedStyle) {
      dir = window.getComputedStyle(document.body, null).getPropertyValue('direction');
    } else {
      dir = (document.body as any).currentStyle.direction;
    }
  }

  return dir === 'ltr';
}

/**
 * Returns whether or not the current device is an iOS device.
 *
 * @return {boolean} Device is an iOS device (i.e. iPod touch/iPhone/iPad).
 */
export function isIOS(): boolean {
    if (typeof window !== 'undefined' && typeof navigator!== 'undefined' ) {
         return /iPad|iPhone|iPod/.test(navigator.userAgent) && !(window as any).MSStream;
    }
    else{
        return false;
    }
}
```

**Comment by arkon** [comment]
@guzuomuse Ah, sorry. I've published v5.0.1 with those.

**Comment by dexbytesmanish** [comment]
Facing this issue with Angular Universal SSR :

ERROR ReferenceError: document is not defined
    at Object.isLTR (/var/www/html/project-angular6/dist/server.js:115630:43)
    at Sidebar.module.exports.Sidebar._normalizePosition (/var/www/html/project-angular6/dist/server.js:109255:27)
    at new Sidebar (/var/www/html/project-angular6/dist/server.js:108715:14)
    at createClass (/var/www/html/project-angular6/dist/server.js:9689:20)
    at createDirectiveInstance (/var/www/html/project-angular6/dist/server.js:9572:20)
    at createViewNodes (/var/www/html/project-angular6/dist/server.js:10792:36)
    at Object.createEmbeddedView (/var/www/html/project-angular6/dist/server.js:10700:5)
    at TemplateRef_.module.exports.TemplateRef_.createEmbeddedView (/var/www/html/project-angular6/dist/server.js:9287:38)
    at ViewContainerRef_.module.exports.ViewContainerRef_.createEmbeddedView (/var/www/html/project-angular6/dist/server.js:9153:35)
    at NgIf.module.exports.NgIf._updateView (/var/www/html/project-angular6/dist/server.js:47980:45)
